src/osd/sdl/sdlmain.cpp

```diff
@@ -452,15 +452,16 @@ void sdl_osd_interface::init(running_machine &machine)
 			//osd_setenv(SDLENV_RENDERDRIVER, stemp, 1);
 			SDL_SetHint(SDL_HINT_RENDER_DRIVER, stemp);
 		}
+// Disabled for now. Don't try to be smarter than SDL. opengl renderdriver looks broken on windows
+#if 0 && defined(SDLMAME_WIN32)
```

Comment:
This seems wrong.  I just tested with MAME 0243 built against MinGW SDL 2.0.14 and the OpenGL renderer works fine.

Reply:
Thanks a lot for testing. Than this is an issue with VirtualBox opengl. Still, I am convinced we should remove this. It was added years ago.